go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/celestiaorg/celestia-node
 
-go 1.22.0
+go 1.22.3
 
 require (
 	cosmossdk.io/errors v1.0.1
 	cosmossdk.io/math v1.3.0
 	github.com/BurntSushi/toml v1.3.2
 	github.com/alecthomas/jsonschema v0.0.0-20220216202328-9eeeec9d044b
 	github.com/benbjohnson/clock v1.3.5
-	github.com/celestiaorg/celestia-app v1.9.0
+	github.com/celestiaorg/celestia-app v1.10.1
 	github.com/celestiaorg/go-fraud v0.2.1
 	github.com/celestiaorg/go-header v0.6.1
 	github.com/celestiaorg/go-libp2p-messenger v0.2.0
-	github.com/celestiaorg/nmt v0.21.0
+	github.com/celestiaorg/nmt v0.21.1-0.20240602221058-a81b748b6f51 // TODO replace with release when ready
```

Comment:
will be replaced once we release nmt

Reply:
[3b2680b](https://github.com/celestiaorg/celestia-node/pull/3470/commits/3b2680b6bee005aa82fdc5a9ced69b0a77aa46ee)